packages/utils/src/network/determineCrossOrigin.ts

```diff
@@ -23,16 +19,7 @@ export function determineCrossOrigin(url: string, loc: Location = globalThis.loc
     // default is window.location
     loc = loc || globalThis.location;
 
-    if (!tempAnchor)
-    {
-        tempAnchor = document.createElement('a');
-    }
-
-    // let the browser determine the full href for the url of this resource and then
-    // parse with the node url lib, we can't use the properties of the anchor element
-    // because they don't work in IE9 :(
-    tempAnchor.href = url;
-    const parsedUrl = _url.parse(tempAnchor.href);
+    const parsedUrl = new URL(url, document.baseURI);
 
     const samePort = (!parsedUrl.port && loc.port === '') || (parsedUrl.port === loc.port);
```

Comment:
The `port` of `URL` would be `''` instead of `null` (for package 'url') if port is absent in the URL, so we don't need special treatment for this anymore.

Reply:
Good observation. Done.